po/ja.po

```diff
@@ -8695,22 +8695,23 @@ msgid "b\"hello\""
 msgstr ""
 
 #: src/welcome-day-4.md
-#, fuzzy
 msgid "Welcome to Day 4"
-msgstr "Day 1へようこそ"
```

Comment:
I don't know where to comment, but it shows up with "Day 1 : AM" in my browser. Do you have any idea why?